### PR TITLE
fix: correctly set `anonymousId` and `userId` for heartbeat events

### PR DESCRIPTION
**Pull request by jmorganca**

### CODE REVIEW

File: internal/server/telemetry.go [hunk 1]
```diff
@@ -90,7 +90,28 @@ func (t *Telemetry) EnqueueHeartbeat() {
...
 
-	t.Event("heartbeat", "", map[string]interface{}{
+	first, err := data.ListIdentities(t.db, &models.Pagination{Limit: 1}, data.NotName(models.InternalInfraConnectorIdentityName))
+	if err != nil {
+		logging.Debugf("%s", err.Error())
+	}
+
+	var admin models.Identity
+	if len(first) > 0 {
+		admin = first[0]
```
Comment (BruceMacD) [inline internal/server/telemetry.go:100]: Is this guaranteed to be the admin? What if Infra was setup using a config file?

Comment (dnephin) [reply internal/server/telemetry.go:100]: The list of `ListIdentities` is most likely alphabetical as well, so probably not the admin.

I think we likely need to query for grants and find the `infra/admin` grant with the earliest created time.

Comment (jmorganca) [reply internal/server/telemetry.go]: Thanks for the review. This should be ready for another quick look. @BruceMacD @dnephin

File: internal/server/telemetry.go [hunk 2]
```diff
@@ -90,7 +90,48 @@ func (t *Telemetry) EnqueueHeartbeat() {
...
-	t.Event("heartbeat", "", map[string]interface{}{
+	admins, err := data.ListGrants(
+		t.db,
+		&models.Pagination{Limit: 1},
+		data.ByResource(access.ResourceInfraAPI),
+		data.ByPrivilege(models.InfraAdminRole),
+		data.ByIsIdentity(),
+	)
+	if err != nil {
+		logging.Debugf("%s", err.Error())
```
Comment (mxyng) [inline internal/server/telemetry.go:101]: Should try to use the zerolog interface directly when logging errors:

```go
logging.L.Debug().Err(err).Msg("")
```

**Review by mxyng — APPROVED** [review]
LGTM assuming it's ok to get one admin, not a specific admin. An improvement maybe to sort on created by to get the first admin user

**Review by dnephin — APPROVED** [review]

**Pull request merged**